src/main/java/core/basesyntax/service/operation/OperationHandler.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.operation;
+
+import core.basesyntax.dao.OperationDao;
+import core.basesyntax.dao.impl.OperationDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationHandler {
+    OperationDao operationDao = new OperationDaoImpl();
+    void operationHandler(FruitTransaction fruitTransaction);
```

Comment:
void method names should be verbs as we perform some actions there and not expecting any result
```suggestion
    void processOperation(FruitTransaction fruitTransaction);
```